src/main/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspect.java

```diff
@@ -82,9 +84,9 @@ private void verifyAllocatedToTeam(UUID caseUUID, UUID stageUUID) {
     }
 
     private boolean proceedIfUserTeamIsAdminForCaseType(UUID caseUUID) {
-        String stageType = caseworkClient.getCase(caseUUID).getType();
+        String shortCode = caseUUID.toString().substring(34);
```

Comment:
This still feels like a hack, can we add something to review this...

Reply:
there is an endpoint on casework we could add to workflow, but that endpoint calls the info service one. so we can unpick it as one task.